CONTRIBUTING_MODELS.md

```diff
@@ -1,57 +1,104 @@
 # Model Contribution Guide
 
-KerasHub has a plethora of pre-trained large language models
-ranging from BERT to OPT. We are always looking for more models and are always
-open to contributions!
+KerasHub has a plethora of pre-trained large language models ranging from BERT 
+to OPT. We are always looking for more models and are always open to 
+contributions!
 
-In this guide, we will walk you through the steps one needs to take in order to
-contribute a new pre-trained model to KerasHub. For illustration purposes, let's
-assume that you want to contribute the DistilBERT model. Before we dive in, we encourage you to go through
-[our getting started guide](https://keras.io/guides/keras_nlp/getting_started/)
-for an introduction to the library, and our
-[contribution guide](https://github.com/keras-team/keras-hub/blob/master/CONTRIBUTING.md).
+In this guide, we will walk you through the steps needed to contribute a new 
+pre-trained model to KerasHub. For illustration purposes, let's assume that you 
+want to contribute the DistilBERT model. Before we dive in, we encourage you to 
+go through our [Getting Started Guide](https://keras.io/guides/keras_nlp/getting_started/) 
+for an introduction to the library, and our [Contribution Guide](https://github.com/keras-team/keras-hub/blob/master/CONTRIBUTING.md).
+
+---
 
 ## Checklist
 
 This to-do list is a brief outline of how a model can be contributed.
 Keep this checklist handy!
 
-### Step 1: Open an issue/find an issue
+### Step 1: Open an Issue or Find an Issue
 
 - [ ] Open an issue or find an issue to contribute a backbone model.
 
-### Step 2: PR #1 - Add XXBackbone
+### Step 2: PR #1 - Model Folder
```

Comment:
Do we want this many PRs? Might be better to ask for a single PR with backbone, initial task, and colab showing usage and results. Less likely to have incomplete model contributions.

I'd say definitely not this, we don't want people opening up PRs just to create empty model folders. That is just more review for us (with nothing of value in the PR).